_includes/posts-paginated.html

```diff
@@ -7,25 +7,37 @@
 
 {% for post in posts %}
 
-    {#- Newsroom contains select Blog posts.
-        These Newsroom Blog posts can be identified by a special
-        `display_in_newsroom` property.
-        When a Blog post is listed in Newsroom we should ignore its category
-        name and instead categorize it as "Blog". #}
-    {%- if vars.path == '/newsroom/' and post.custom_fields.display_in_newsroom %}
-        {% set category = 'Blog' %}
-    {% elif post.category.0 %}
-        {% set category = post.category.0 %}
+    {#- Blog uses a different category field ('blog_category')
+        so on that page we'll need to tell category_slug to use that field
+        rather than the normal 'category' #}
+    {%- if vars.path == '/blog/' %}
+        {% set category = post.blog_category %}
+        {% set use_blog_category = True %}
+    {% elif post.category %}
+        {% set category = post.category %}
+        {% set use_blog_category = False %}
     {% endif -%}
     {%- import "category-slug.html" as category_slug %}
-    {%- set slug = category_slug.render(category, vars.path, 'meta-header_left') -%}
 
     <article class="post-preview">
         <div class="meta-header">
             <span class="date meta-header_right">
                 {{ post.date|date('%b %d, %Y') }}
             </span>
-            {{ slug if category else '&nbsp;' }}
+            {% if not category %}
+                {{ '&nbsp;'|safe }}
+            {% else %}
+                {% for cat in category %}
```

Comment:
We'll probably need to wrap each category with it's pipe if needed in a span and then set a style to make it visually a block elem on smaller screens.